### ISSUE TITLE
Add options state

### ISSUE THREAD

**Issue by paulgoetze** [kennethlarsen/thecatgame#16]
We should have an options page, where you can  

* switch on/off the sound
* check/uncheck skipping the intro (there is a localStorage value already set on closing the intro state, so this would be for reenabling it)

All option values should read/write their values from/to the localStorage (any better idea @kennethlarsen or would this be the way to go for global state?)

The option page should be accessible from the page title state and probably by a hotkey while playing (like a pause state or the like).

**Comment by kennethlarsen** [comment]
Good idea. I think localStorage is best for this. Don't think Phaser has any alternative for this.
